xchange-gemini/src/main/java/org/knowm/xchange/gemini/v1/service/GeminiTradeServiceRaw.java

```diff
@@ -134,6 +132,47 @@ public GeminiOrderStatusResponse getGeminiOrderStatus(String orderId) throws IOE
     }
   }
 
+  public GeminiOrderStatusResponse getGeminiOrderStatus(OrderQueryParams params) throws IOException {
+    Long orderId = null;
+    String clientOrderId = null;
+    boolean includeTrades = false;
+    String account = null;
+
+    if (params instanceof GeminiTradeService.GeminiOrderQueryParams) {
+      orderId = Long.valueOf(params.getOrderId());
+
+      // If both types of ids are included in the params, default to using orderId
+      if (orderId == null) {
+        clientOrderId = ((GeminiTradeService.GeminiOrderQueryParams) params).getClientOrderId();
+      }
+
+      includeTrades = ((GeminiTradeService.GeminiOrderQueryParams) params).isIncludeTrades();
+      account = ((GeminiTradeService.GeminiOrderQueryParams) params).getAccount();
+    }
+
+    try {
+      GeminiOrderStatusResponse orderStatus =
```

Comment:
just return inline, no need to set this var

Reply:
Done